addons/project/static/src/views/project_task_calendar/project_task_calendar_controller.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { CalendarController } from "@web/views/calendar/calendar_controller";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+
+export class ProjectTaskCalendarController extends CalendarController {
+    setup() {
+        super.setup(...arguments);
+        this.displayName += this.env._t(" - Tasks by Deadline");
+    }
+
+    deleteRecord(record) {
+        if  (record.rawRecord.subtask_count > 0) {
+            this.displayDialog(ConfirmationDialog, {
+                title: this.env._t("Confirmation"),
```

Comment:
Isn't this the default title? But maybe aju want the title to be something like "Super-task deletion" or something like that. I know she likes to give a title to dialogs :shrug: